test/fixtures/drawing/draw-correct-order-if-undefined-values.js

```diff
@@ -0,0 +1,60 @@
+// https://github.com/chartjs/chartjs-plugin-datalabels/issues/319
+
+export default {

```

Comment:
If you look at the other tests, there are very minimal and focus on what need to be tested. That's to improve test stability across different platforms but also when upgrading Chart.js. So I would suggest to get rid the options that doesn't bring any value:

- make label square (i.e. no border radius)
- hide the label text (i.e. `{ font: { size: 0 } }`)
- hide lines between data points

At the end, the generated PNG should only show red squares on top of blue squares

Reply:
done